postgresql/_database.sls

```diff
@@ -39,6 +39,18 @@ postgresql_database_{{ svr_name|default('localhost') }}_{{ database_name }}:
     - user: root
     {%- endif %}
 
+{%- if database.init is defined and database.init.queries is defined %}
+{%- for query in database.init.get('queries', []) %}
+postgresql_database_{{ svr_name|default('localhost') }}_{{ database.init.get('maintenance_db', database_name) }}_{{ loop.index }}:
+  cmd.run:
```

Comment:
Should be cmd.wait and watch `postgres_database.present` to happen only on db creation.